ImperatorToCK3/Source/Imperator/Characters/Character.cpp

```diff
@@ -44,6 +70,34 @@ void ImperatorWorld::Character::registerKeys()
 		const commonItems::singleInt familyInt(theStream);
 		family = std::pair(familyInt.getInt(), nullptr);
 	});
+	registerKeyword("dna", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
Putting on my design hat, dna is sufficiently complex that it should be moved to its own class. It also needs a more robust test suite.

Reply:
✔️ Added a `CharacterPortraitData` class.